database/migrations/2019_03_21_111809_add_thumbnail_to_news.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddThumbnailToNews extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('news_items',function(Blueprint $table){
+            $table->string('thumbnail_url')->nullable();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        Schema::table('news_items',function(Blueprint $table){
+            $table->dropColumn('thumbnail_url')->nullable();
```

Comment:
`nullable()` kan hier weg, durf niet te zeggen of het een error oplevert eigenlijk.

Reply:
Inderdaad, goed gezien! Zal hem weghalen voordat ik merge.

Reply:
Gefixt!